tests/components/unifi/test_config_flow.py

```diff
@@ -459,17 +460,25 @@ async def test_advanced_option_flow(
     assert result["type"] == data_entry_flow.FlowResultType.FORM
     assert result["step_id"] == "device_tracker"
     assert not result["last_step"]
-    assert set(
-        result["data_schema"].schema[CONF_SSID_FILTER].options.keys()
-    ).intersection(("SSID 1", "SSID 2", "SSID 2_IOT", "SSID 3"))
-
+    ssids_after_init = result["data_schema"].schema[CONF_SSID_FILTER].options.keys()
+    assert set(ssids_after_init).intersection(
+        ("SSID 1", "SSID 2", "SSID 2_IOT", "SSID 3", "SSID 4")
+    )
+    assert list(ssids_after_init) == [
+        "",
```

Comment:
Just noticed this, is that one really relevant? What warranted this extra control?

Reply:
The previous check did not check the returned SSIDs in detail, though it was not checking for unexpected entries like the ones with suffix "None". I thought it would be more robust to check the response that way. If you think it's too much details we can leave it